raster/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM continuumio/anaconda
```

Comment:
We don't need dockerfile this year. Please remove to keep repository clean. Thanks! :)